app/Models/Transformers/StopTransformer.php

```diff
@@ -10,7 +10,7 @@ class StopTransformer extends TransformerAbstract
     public function transform(TwillModelContract $stop): array
     {
         return [
-            'object' => "$stop->object_type:$stop->object_id",
+            'object' => class_basename($stop->selector?->object::class) . ':' . $stop->object_id,
```

Comment:
You'll also need to update the `ObjectTransformer`, which currently lowercases the class name:
https://github.com/art-institute-of-chicago/aic-mobile-cms/blob/develop/app/Models/Transformers/ObjectTransformer.php#L21

Reply:
Ah, good to know. I'll update this PR to match what the `ObjectTransformer` is already doing.